analyzers/.sonarlint/sonaranalyzer-dotnet/CSharp/SonarLint.xml

```diff
@@ -171,7 +171,7 @@
       <Key>S3776</Key>
       <Parameters>
         <Parameter>
-          <Key>propertyThreshold</Key>
+          <Key>propertyThreshold </Key>
```

Comment:
This should not be here, I've noticed that when playing with the QP.
Fix is on the way #5897
Please revert this change in all 3 PRs